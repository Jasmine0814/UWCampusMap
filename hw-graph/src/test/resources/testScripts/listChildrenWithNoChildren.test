# test whether a node without children can be listed.

CreateGraph A
AddNode A n1
ListChildren A n1