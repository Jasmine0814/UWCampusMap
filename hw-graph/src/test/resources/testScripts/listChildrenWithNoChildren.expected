# test whether a node without children can be listed.

created graph A
added node n1 to A
the children of n1 in A are: